.circleci/config.yml

```diff
@@ -2,34 +2,45 @@
 #
 # Check https://hub.docker.com/r/circleci/node/ for more details
 #
-version: 2
-jobs:
-  build:
-    docker:
-      - image: circleci/node:10.11-browsers
 
-    working_directory: ~/sinopia_profile_editor
+defaults: &defaults
+  docker:
+    - image: circleci/node:10.11-browsers
+  working_directory: ~/sinopia_profile_editor
 
+version: 2
+jobs:
+  # Maintains cache of 2 sets of node_modules: production and CI
+  dependencies:
+    <<: *defaults
     steps:
       - checkout
-      - run:
-          name: Install latest npm
-          command: 'sudo npm install -g npm@latest'
       - restore_cache:
-          key: dependency-cache-{{ checksum "package.json" }}
-      - run:
-          name: npm install
-          command: npm install
+          key: dependency-cache-prod-{{ checksum "package.json" }}
+      - run: npm install --only=production
       - save_cache:
-          key: dependency-cache-{{ checksum "package.json" }}
+          key: dependency-cache-prod-{{ checksum "package.json" }}
           paths:
             - node_modules
-      - run:
-          name: eslint
-          command: npm run eslint
-      - run:
-          name: build with grunt
```

Comment:
It obscures the command actually being run in Circle CI making debugging more difficult.

Reply:
I think installing grunt with -g as part of the build was also from copy-pasta of the sample configs.  

I slightly prefer using the npm script and the grunt in our own installed modules -- if we change the grunt stuff in our npm script then it would be changed in circleci without us having to remember to do it.   

So I would prefer removing the -g install of grunt and keeping  npm run grunt-dev;   this will be more easily reproduced on our laptops for consistency.